src/utils/processEvents.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (logger) => {
+	process
```

Comment:
I do not like that it is located in utils, because it is app settings and configure stuff. Yes true it should not at to middleware but we has no configure/ folder at the moment and move all app configuration stuff to the app, or to the middleware.